environment/filesystem.go

```diff
@@ -2,10 +2,18 @@ package environment
 
 import (
 	"context"
+	_ "embed"
 	"fmt"
 	"strings"
+
+	"dagger.io/dagger"
+	"dagger.io/dagger/dag"
+	"github.com/dagger/container-use/edit"
 )
 
+// FIXME: See hack where it's used
+const fileUtilsBaseImage = "busybox"
```

Comment:
nit: I think there's another one of this pointing to `alpine` in the codebase, can we merge the two?

nit2: can we pin this to a digest? Otherwise AFAIK buildkit will ping the registry each time to resolve the digest and see if it changed compared to local